README.md

```diff
@@ -1,75 +1,328 @@
+
+```
+kubectl config use-context k3d-adelphi
+```
+
+You can now check the status of the cluster using kubectl:
+
+```
+kubectl get nodes
+```
+
+It should provide the status of the nodes specified in the configuration file:
+
+```
+NAME                   STATUS   ROLES         AGE     VERSION
+k3d-adelphi-server-0   Ready    etcd,master   10m     v1.19.3+k3s2
+k3d-adelphi-server-1   Ready    etcd,master   10m     v1.19.3+k3s2
+k3d-adelphi-server-2   Ready    etcd,master   9m51s   v1.19.3+k3s2
+```
+
+When you are done with the cluster, it can be deleted with:
+
+```
+k3d cluster delete adelphi
+```
+
+## Running Adelphi
+
+With helm installed, a kubernetes cluster deployed, and the kubectl context set appropriately you're now ready to deploy Adelphi.
+
+### Download Source
+
+To get started, you'll need a copy of the Adelphi repo to get access to the helm charts and components.
+
+You can clone it from [GitHub](https://github.com/datastax/adelphi) using [git](https://git-scm.com/):
 
 ```
 git clone https://github.com/datastax/adelphi
-cd adelphi
+```
+
+Or download it manually from [GitHub](https://github.com/datastax/adelphi/archive/master.zip) and extract the contents of the archive.
+
+### Deployment
+
+As mentioned, Adelph is deployed using helm.  Within the Adelphi repository you will find a series of helm charts in the `/helm` directory.  
+
+To deploy Adelphi with the default configuration navigate to the source directory and run:
+
+```
 helm install adelphi helm/adelphi -n cass-operator
 ```
 
-> :warning: This assumes your KUBECONFIG is set to the right k8s cluster and that you have the Helm package manager installed.
-For complete setup instructions, check [Local Setup](#local-setup).
-                                     
-## What is Adelphi?
-It is an automation tool for testing C* OSS that assembles [NoSQLBench](https://github.com/nosqlbench/nosqlbench) and
-[cassandra-diff](https://github.com/apache/cassandra-diff) (with other tools coming soon!).
+> :warning: The command shown above will install the Adelphi components into the `cass-operator` namespace.  Using this namespace is currently **required**.  
+>
+>Please see [#93](https://github.com/datastax/adelphi/issues/93) for more information and follow along as we work to better support custom namespace scoped installations.
+
+This will deploy Adelphi with the default [values](helm/adelphi/values.yaml) configuration.
+
+### Configuration
+
+To control the deployment configuration, many options are made available via the helm charts included with Adelphi.
+
+To learn more about setting values via helm, see their [documentation](https://helm.sh/docs/chart_template_guide/values_files/).
+
+In general, with helm, there are two ways to modify the values provided to a deployment:
+* via the command line using `--set value.path=value`
+* via a `yaml` file via `-f custom.values.yaml`
```

Comment:
It's worth mentioning here that the default configuration is for running locally on k3d.
For running on GKE, Adelphi comes with a custom values YAML that can be used with:

```
helm install adelphi helm/adelphi -n cass-operator -f helm/adelphi/gke-values.yaml
```